src/test/java/com/wikia/webdriver/testcases/forumtests/ForumEditModeTests.java

```diff
@@ -99,5 +103,47 @@ public void adminUserCanMoveBoard() {
     Assertion.assertTrue(beforeMoveDown < manageForum.getBoardPosition(first), "");
     manageForum.clickMoveUp(first);
     Assertion.assertTrue(beforeMoveDown == manageForum.getBoardPosition(first), "");
+  }
+
+  @Test
+  public void clickExplore(){
```

Comment:
All test's names in this class should be more descriptive. This one could be e.g. `clickExploreRedirectsToSpecialImages` or any similar